src/course-unit/hooks.jsx

```diff
@@ -20,8 +20,11 @@ import {
   getLoadingStatus,
   getSavingStatus,
   getSequenceStatus,
+  getCourseUnitEnableCopyPaste, getStaticFileNotices,
```

Comment:
[nit] Place one property per line 
```suggestion
  getCourseUnitEnableCopyPaste,
  getStaticFileNotices,
```

Reply:
Corrected, thanks